husky_gazebo/launch/realsense.launch

```diff
@@ -1,3 +1,5 @@
+<?xml version="1.0"?>
```

Comment:
This is a separate fix and not directly related to this change

Reply:
You're right, reverted back (the requested changes have been made). Can create a separate PR for this fix.